### PR TITLE
cloud-init: i wasn't just twiddling my thumbs

### PR DESCRIPTION
**Pull request by igalic**
mention kernel fixes and debugging i did while waiting for resources

### CODE REVIEW

**Comment by Jehops** [comment]
LGTM.

**Review by lsalvadore — APPROVED** [review]
Thanks @igalic.

Adding @dbaio and @sergio-carlavilla as reviewers for mentor approval as usual.

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc [hunk 1]
```diff
@@ -16,6 +16,10 @@ This quarter has been going very, very slowly, for personal reasons — but also
 I have been trying to link:https://github.com/canonical/cloud-init/pull/2003[port the Infiniband functions].
 This has proven difficult, because it falsified my thesis that man:ifconfig[8] is all that is needed to figure out network interfaces on FreeBSD.
 
+While waiting for resources, I link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269767[debugged a boot panic] and got it fixed: gitref:499171a98c8813e4dc6e085461d5c47750efa555[repository=src].
+This now makes it possible to boot FreeBSD on LXD - cloud-init's CI platform.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:20]: A pair of hyphens: 

> … LXD -- cloud-init's …

– will be rendered as an em dash. 

<https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-replacements>

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc [hunk 2]
```diff
@@ -16,6 +16,10 @@ This quarter has been going very, very slowly, for personal reasons — but also
...
+We still need to fix the link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269823[high CPU problem], but there is already an accepted review: link:https://reviews.freebsd.org/D38898[D38898]
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:21]: high CPU usage problem

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc (hunk 1 above)
Comment (igalic) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:20]:
```suggestion
This now makes it possible to boot FreeBSD on LXD -- cloud-init's CI platform.
```

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc (hunk 2 above)
Comment (igalic) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:21]:
```suggestion
We still need to fix the link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269823[the high CPU usage problem], but there is already an accepted review: link:https://reviews.freebsd.org/D38898[D38898]
```

**Review by igalic — COMMENTED** [review]
add @grahamperrin's suggestions… as suggestions, so i can commit them

**Comment by grahamperrin** [comment]
> … suggestions… as suggestions, …

Oh, this is a neat feature. Thanks!

<https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request> point ⑥, I never noticed that. 

(I rarely look at editing toolbar buttons.)

**Comment by lsalvadore** [comment]
Merged with commit https://github.com/freebsd/freebsd-doc/commit/84dc04b888c9c10a77d33b331e5043fc1539293b. Thanks everyone.

**Pull request closed without merging**